addons/common/functions/fnc_isRemoteControlled.sqf

```diff
@@ -0,0 +1,22 @@
+#include "script_component.hpp"
+/*
+ * Author: NeilZar
+ * Check whether the given unit is currently remote controlled.
+ *
+ * Arguments:
+ * 0: Unit <OBJECT>
+ *
+ * Return Value:
+ * Unit is Remote Controlled <BOOL>
+ *
+ * Example:
+ * [_unit] call zen_common_fnc_isRemoteControlled
+ *
+ * Public: Yes
+ */
+
+params ["_unit"];
+
+if !(_unit isKindOf "CAManBase") exitWith {false};
+
+_unit getVariable ["bis_fnc_moduleremotecontrol_owner", objNull] in allCurators
```

Comment:
Variable is set as the controlling player unit, not the controlling curator logic.